bin/default/alignak.in

```diff
@@ -64,8 +64,12 @@ LIB=$LIB$
 ARBITERCFG="$ETC/daemons/arbiterd.ini"
 
 # location of the alignak configuration file
-# Please update $ETC$ instead of this one.
-ALIGNAKCFG="$ETC/alignak.cfg"
+# Now look if some required variables are pre defined:
+if ! test "$ALIGNAKCFG"
```

Comment:
can you use the [ -z $ALIGNAKCFG ] ? [ is an alias for test, just to keep consitent. Unless you are afraid of some compatibility issue